x-pack/plugins/drilldowns/url_drilldown/public/lib/url_drilldown.tsx

```diff
@@ -75,20 +88,39 @@ export class UrlDrilldown implements Drilldown<Config, ChartActionContext, Actio
     context: ChartActionContext | ActionExecutionContext<ChartActionContext>;
   }> = ({ config, context }) => {
     const [title, setTitle] = React.useState(config.name);
+    const [error, setError] = React.useState<string | undefined>();
     React.useEffect(() => {
-      let unmounted = false;
       const variables = this.getRuntimeVariables(context);
       urlDrilldownCompileUrl(title, variables, false)
         .then((result) => {
-          if (unmounted) return;
           if (title !== result) setTitle(result);
         })
         .catch(() => {});
-      return () => {
-        unmounted = true;
-      };
-    });
-    return <>{title}</>;
+
+      // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
nit: I think it might be sufficient to check that within this hook the effect only runs when title is undefined since we only ever set it's value here, here by removing the need for this comment, wdyt?

Reply:
Looks like initial title is not undefined, but is a raw value :( 
it makes the potential check a bit less clear and reliable .